test/conformance/parallel/parallel_suite_test.go

```diff
@@ -0,0 +1,51 @@
+//go:build !unittests
+// +build !unittests
+
+package test
+
+import (
+	"flag"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+
+	ptp "github.com/openshift/ptp-operator/test/conformance/common"
+	"github.com/openshift/ptp-operator/test/utils"
+	"github.com/openshift/ptp-operator/test/utils/clean"
+	testclient "github.com/openshift/ptp-operator/test/utils/client"
+	"github.com/openshift/ptp-operator/test/utils/testconfig"
+)
+
+var junitPath *string
+var DeletePtpConfig bool
+
+func init() {
+	junitPath = flag.String("junit", "junit.xml", "the path for the junit format report")
+}
+
+func TestTest(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "PTP e2e tests : Parallel")
+}
+
+var _ = SynchronizedBeforeSuite(func() []byte {
+	// Run on all Ginkgo nodes
+	logrus.Info("Executed from parallel suite")
+	testclient.Client = testclient.New("")
+	Expect(testclient.Client).NotTo(BeNil())
+
+	testconfig.CreatePtpConfigurations()
```

Comment:
For this reason, you need to rename the configuration to specific name. configuration==ptpTestCaseConfig